generators/add-action/index.js

```diff
@@ -29,9 +30,9 @@ const sdkCodeToTitle = {
   [sdkCodes.target]: 'Adobe Target',
   [sdkCodes.analytics]: 'Adobe Analytics',
   [sdkCodes.campaign]: 'Adobe Campaign Standard',
+  [sdkCodes.assetCompute]: 'Asset Compute',
```

Comment:
This pops up in the generator selection when asked for "Which type of sample actions do you want to create?"

Should we name this `Asset Compute Worker` or `Asset Compute Custom Worker` instead? That would describe it better IMO.

cc @sarahxxu @purplecabbage @meryllblanchet

Reply:
This is up to you, that text is part of this pr.
`Asset Compute Custom Worker` sounds better to me. You also mentioned possibly adding (requires provisioning) or something.

Reply:
@alexkli do you voluntarily omit `AEM` in front of `Asset Compute Custom Worker`?
Otherwise LGTM.